lib/Db/FrameMapper.php

```diff
@@ -216,6 +219,9 @@ public function updateFrame(Frame $frame, string $name, string $userUid, int $al
     $frame->setStartDayAt($startDayAt);
     $frame->setEndDayAt($endDayAt);
     $frame->setShowPhotoTimestamp($showPhotoTimestamp);
+    $frame->setStyleFill($styleFill);
+    $frame->setStyleBackgroundColor($styleBackgroundColor);
```

Comment:
I would prefer `styleBackgroundColor` to default to null (not having been actively set). I think there's a difference between using the "default" theme and setting a background color.